lib/kapnismology/smoke_test.rb

```diff
@@ -12,6 +12,10 @@ class SmokeTest
     RUNTIME_TAG = 'runtime'.freeze
     DEFAULT_TAGS = [DEPLOYMENT_TAG, RUNTIME_TAG].freeze
 
+    def initialize
```

Comment:
The problem is that classes inheriting from SmokeTest are not required to do `super` on their `initialize` if they happer to have an `initialize`
So there will be a bug on that case, and adding that restriction to the classes inheriting just to avoid the warning ...